src/codewhisperer/service/transformationHubViewProvider.ts

```diff
@@ -237,9 +250,9 @@ export class TransformationHubViewProvider implements vscode.WebviewViewProvider
         if (stepStatus === StepProgress.Succeeded) {
             return `<span style="color: green"> ✓ </span>`
         } else if (stepStatus === StepProgress.Pending) {
-            return `<span> <i class="fas fa-spinner fa-spin"></i> </span>` // TODO: switch from FA to native VSCode icons
+            return `<span style="color: grey" class="spinner"> ↻ </span>`
```

Comment:
Question: Is there a better way to show this spinner. It looks like you copy/pasted the unicode character itself from the VSCode website?

Docs -> https://code.visualstudio.com/api/references/icons-in-labels#animation

Can you do something like this example -> https://github.com/microsoft/vscode-extension-samples/blob/22d5639ff5c1d88f144c057fc3d29cc9dfd99d62/webview-codicons-sample/src/extension.ts#L34

Question: Not sure if the "codicon" library is the native icons. Looks like those are not for webviews from the docs say you CANT use the native icons in webview:
<img width="793" alt="Screenshot 2024-01-09 at 10 51 11 AM" src="https://github.com/aws/aws-toolkit-vscode/assets/38665452/1162d176-d861-42f3-be0b-8d230650fcb8" />

Reply:
here's an example of a codicon being used in a webview: https://github.com/aws/aws-toolkit-vscode/blob/a11d5db448b9697adb8e3a0038be248e1e3e1660/src/codewhisperer/views/securityPanelViewProvider.ts#L51-L53

Reply:
Yes the icon itself is copied/pasted. I initially had it like the example you linked but we don't want to be going into `node_modules` since it doesn't exist in the VSIX.

Reply:
@justinmk3 I am unable to find an `icons.css` file locally. Also what is the concern with copying/pasting the icon?

Reply:
<img width="217" alt="Screenshot 2024-01-09 at 5 47 09 PM" src="https://github.com/aws/aws-toolkit-vscode/assets/60020664/ce7e37fb-1f5b-402a-a450-26c1dae9973e">

Copied the code exactly from that example and for some reason only a small fraction of the icon appears. Tried altering the CSS in various ways but unable to resolve it. If this is a significant problem then I can make a SIM for it but I would prefer to get the other important changes in this PR out sooner.